jlm/llvm/ir/operators/lambda.hpp

```diff
@@ -146,8 +142,7 @@ class node final : public jlm::rvsdg::structural_node
   using fctargument_range = jlm::util::iterator_range<fctargiterator>;
   using fctargument_constrange = jlm::util::iterator_range<fctargconstiterator>;
 
-  using ctxvar_range = jlm::util::iterator_range<cviterator>;
-  using ctxvar_constrange = jlm::util::iterator_range<cvconstiterator>;
+  using ctxvar_constrange = jlm::util::iterator_range<rvsdg::node_const_input_iterator>;
```

Comment:
I want to get away from the the `ctxvar` naming. It is not very intuitive.

`using LambdaInputConstRange = ...`

Reply:
Sounds good, in principle I would say this does not even need a typename alias -- knowing that it returns a list of inputs ought to be enough. Eventually, a generic `NodeInputRange` to represent arbitrary lists of nodes would most likely emerge as the generic thing in all places where currently specific range types are used.